src/utils/getVisiblePeople.ts

```diff
@@ -0,0 +1,55 @@
+import { Person } from '../types';
+
+export const getVisiblePeople = (
+  people: Person[],
+  sex: string | null,
+  query: string | null,
+  centuries: string[] | null,
+  sortParam: string,
+  orderParam: string,
+) => {
+  let peopleCopy = [...people];
+
+  if (sex) {
+    peopleCopy = peopleCopy.filter(person => person.sex === sex);
+  }
+
+  if (query) {
+    peopleCopy = peopleCopy.filter(person => {
+      return (
+        person.name.toLowerCase().includes(query) ||
+        person.motherName?.toLowerCase().includes(query) ||
+        person.fatherName?.toLowerCase().includes(query)
```

Comment:
let's create separated function for `.toLowerCase().includes(query)` and reuse in these cases